### DISCUSSION TITLE
Check if it is possible to create a redirect on the floss manuals site to point to the new name or revert the change of the link

### DISCUSSION THREAD

**Discussion by abreubacelar** [pure-data/pddp#53] [General]
@porres renamed the floss manuals from pure-data to pd-extended

changing the main link from

https://en.flossmanuals.net/pure-data/_info/ 

to https://en.flossmanuals.net/pd-extended/_info/

the change is welcome and was needed to reinforce that the manual was outdated, the problem is that everywhere on the internet the links are now broken, so for example, on puredata.info http://puredata.info/docs/ListOfPdExternals , the list is about pd-extended (and need to be updated also to reinforce that, but is is another issue, and easy to solve), but the reference of the original source is broken, which i think it is bad...

what we could do? I think it is important to keep the original link so old pages about pd-extended don't forward people to 404 pages... but indeed the book itself shouldn't be "pure-data"

should we see if it is possible to create redirects or maybe just revert the original link but keeping the new name of the book?

**Comment by emviveros** [comment]
Maybe we can only archive it in wayback machine. To do it:
1. Revert pd-extended floss to https://en.flossmanuals.net/pure-data/_info/
2. Archive it in wayback machine: https://archive.org/web/
3. Once archived, re-revert pd-extended floss to: https://en.flossmanuals.net/pd-extended/_info/

And the pd-extended rest in peace.

**Comment by porres** [comment]
working with floss is not that easy as floss people need to be involved... and right now all links just seem dead... I still think floss may be some source of confusion and I would like to make it even clearer that it is an old archive document.

anyway, the puredata.info entried should be updated.

**Comment by porres** [comment]
closing this in favor of a new issue https://github.com/pure-data/pddp/issues/224
